pack/config/whitelist.txt

```diff
@@ -76,6 +80,7 @@ ffxiv*                       || Final Fantasy XIV Launcher or Configurator
 FirefallClient               || Firefall
 FoxGame-Win32-Shipping       || Blacklight: Retribution
 fr-041_debris                || "Debris" by Farbrausch
+Game                         || Gurumin: A Monstrous Adventure
```

Comment:
Wouldn't this conflict with other games that have this exact exe name, like all RPGMaker games?